Doc/taipy#1696 - Update Config.checker documentation (#1096)
Avaiga/taipy-doc: docs/manuals/userman/configuration/config-checker.md

- Depending on the `storage_type` value of a `DataNodeConfig^`, an `ERROR` issue is created if a specific required
  property is missing.
- An `ERROR` issue is created if one of the `inputs` and `outputs` parameters of a `TaskConfig^` does not correspond

>   to a `DataNodeConfig` or a list of `DataNodeConfig^`.

Reviewer: ```suggestion
  to a `DataNodeConfig^`.
```

Author: The input and output can be a list right?

Reviewer: Oh !

I interpreted "if one of the `inputs` and `outputs` parameters" as "if one of the items of the lists"...

Nevermind then.

- A `WARNING` issue is created if a `TaskConfig^` has no input and no output.
- An `ERROR` issue is created if any `DataNodeConfig^` of the `inputs` or `outputs` has the same name as one of the
  attributes or properties of the `TaskConfig^`.
- An `ERROR` issue is created if the `function` parameter of a `TaskConfig^` is not a callable function.
- An `ERROR` issue is created if one of the tasks provided in the `sequences` parameters does not does not exist in
  the corresponding list of tasks in `tasks` parameter of the `ScenarioConfig^`.

> - An `ERROR` issue is created if a `ScenarioConfig^` has no task configuration defined or does not correspond to a
>   `TaskConfig` or a list of `TaskConfig^`.

Reviewer: I don't get the second part of this sentence.
An error issue is created if a scenario config ... does not correspond to a task config or a list of task config.

I would split the line in two.

```suggestion
- An `ERROR` issue is created if a `ScenarioConfig^` has no task configuration defined.
- An `ERROR` issue is created if any item of a `ScenarioConfig^` task, does not correspond to a `TaskConfig^`.
```

Author: ```suggestion
- An `ERROR` issue is created if a `ScenarioConfig^` has no task configuration defined.
- An `ERROR` issue is created if any item of a `ScenarioConfig^` task list does not 
  correspond to a `TaskConfig^`.
```

Your suggestion has a redundant comma. Otherwise, I agree

> - A `ERROR` issue is created if a `ScenarioConfig^` has no additional data node configuration defined or an additional
>   does not correspond to a `DataNodeConfig` or a list of `DataNodeConfig^`.

Reviewer: ```suggestion
- A `ERROR` issue is created if any additional data node of a `ScenarioConfig^` does not
  correspond to a `DataNodeConfig^`.
```

Author: We need to specify "any additional data node" of a ScenarioConfig. Without the ScenarioConfig, the sentence doesn't make sense to me
